packages/user-interface/src/utils/get-current-navigation-page.ts

```diff
@@ -14,10 +14,13 @@ export const getCurrentNavigationPage = (
   matches: UIMatch[],
   navigationItems: NavigationItem[][],
 ) => {
-  while (matches.length > 0) {
+  const matchesCopy = [...matches];
+
+  for (let i = 0; i < matchesCopy.length - 1; i++) {
     const match = matches.pop();
 
     if (!match) continue;
+    if (i === matchesCopy.length - 1 && match.pathname === "/") continue;
 
```

Comment:
Ik denk dat dit iets efficienter is:

```suggestion
  const matchesSize = matchesCopy.length - 1;

  for (let i = 0; i < matchesSize; i++) {
    const match = matches.pop();

    if (!match) continue;
    if (i === matchesSize && match.pathname === "/") continue;

```